.github/workflows/ci.yml

```diff
@@ -728,20 +727,23 @@ jobs:
         python -m pip install --upgrade pip
         python -m pip install pytest
         python -m pip install pytest-xdist
+        python -m pip install numba
         echo ::endgroup::
+        echo ::group::Run complete test suite
+        python -m pytest -sv | tee complete_testrun.log 2>&1
         echo ::group::Crashing Test Logs
         # See if we don't have a crash that went away
         # Comment out all xfails but the ones that have a run=False condition.
         find . -name "*.py" -exec sed -i '/run=False/!s/^ *@mark.xfail\(.*\)/#&/' {} \;
-        python -m pytest -n 1 -m "xfail" --runxfail -sv --max-worker-restart 512 | tee test_crashed.log 2>&1
+        python -m pytest -m "xfail" --runxfail -sv | tee test_crashed.log 2>&1 || true
```

Comment:
This change is not good, it will stop the tests at the first crash... Do we want that?

Reply:
I seem to have missed that, it has been added back